tests/_tests_files/test_integrator_wire_plugin_dependency_provider.php

```diff
@@ -3,13 +3,14 @@
 namespace Pyz\Zed\TestIntegratorWirePlugin;
 
 use Spryker\Zed\TestIntegratorWirePlugin\Communication\Plugin\TestIntegratorWirePlugin;
+use Spryker\Zed\TestIntegratorWirePlugin\TestIntegratorWirePluginConfig;
 
 class TestIntegratorWirePluginDependencyProvider
 {
     public function getTestPlugins() : array
     {
         return [
-            new TestIntegratorWirePlugin(),
+            TestIntegratorWirePluginConfig::TEST_INTEGRATOR_WIRE_PLUGIN => new TestIntegratorWirePlugin(),
```

Comment:
Isn't it replacing the original test case? Can you add an additional one rather than change the existing one?